Sources/View/ReusableView.swift

```diff
@@ -20,15 +20,19 @@ public extension ReusableView where Self: UITableViewCell {
     init() {
         self.init(style: .default, reuseIdentifier: Self.reuseIdentifier)
 
-        setupLayout()
+        setupSubviews()
```

Comment:
As a verb, I would change all methods from `setup[...]` to `setUp[...]`.
What do you think?